### ISSUE TITLE
bug export layout pdf

### ISSUE THREAD

**Issue by pmercier125** [academic-moodle-cooperation/moodle-local_assignsubmission_download#8]
hello, the table is not complete when we export to pdf (portrait or landscape) ; Tests on 3.11 and 4.1 with adaptable and boost, thematic format, assignment submitted or not,
[BIOL-J201_-_Introduction__ltude_des_plantes_mdicinales_-_202223-Herbier_note_-_premire_session (5).pdf](https://github.com/academic-moodle-cooperation/moodle-local_assignsubmission_download/files/11627382/BIOL-J201_-_Introduction__ltude_des_plantes_mdicinales_-_202223-Herbier_note_-_premire_session.5.pdf)

**Comment by goggo24** [comment]
Hmm ... have you checked "optimum" when exporting? If not, can you try it, whether the result changes. Thanks. Can you try another format too - that would be great.

**Comment by pmercier125** [comment]
yes absolutely: optimum checked or not, no difference unfortunately (3.11 and 4.1)

**Comment by gergelyrakoczi** [comment]
@pmercier125, Can you provide me with a screenshot where you experience the issue? To be honest, I do not see a relation between the bug report and the plugin. Thank you.
